src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/ConfigurationGeneral.xaml

```diff
@@ -123,30 +116,24 @@
                   Visible="False" />
 
   <BoolProperty Name="Optimize"
-                Description="Whether the compiler should optimize output."
-                DisplayName="Optimize" />
+                Visible="False" />
 
   <StringProperty Name="OutputName"
-                  DisplayName="Output Name" />
+                  Visible="False" />
 
   <StringProperty Name="OutDir"
                   Visible="False" />
 
   <StringProperty Name="OutputPath"
-                  DisplayName="Output Path" />
+                  Visible="False" />
 
   <EnumProperty Name="OutputType"
-                DisplayName="Output Type" >
-    <EnumValue Name="Library"
-               DisplayName="Class Library" />
-    <EnumValue Name="Exe"
-               DisplayName="Console Application" />
-    <EnumValue Name="WinExe"
-               DisplayName="Windows Application" />
-    <EnumValue Name="AppContainerExe"
-               DisplayName="Windows Store app" />
-    <EnumValue Name="WinMDObj"
-               DisplayName="WinMD" />
+                Visible="False" >
+    <EnumValue Name="Library" />
+    <EnumValue Name="Exe" />
+    <EnumValue Name="WinExe" />
+    <EnumValue Name="AppContainerExe" />
+    <EnumValue Name="WinMDObj" />
```

Comment:
This loses some information. It's not clear from the name that `AppContainerExe` is a Windows Store app.

```suggestion
    <EnumValue Name="Library" />
    <EnumValue Name="Exe" />                 <!-- Console app -->
    <EnumValue Name="WinExe" />
    <EnumValue Name="AppContainerExe" />     <!-- Windows Store app -->
    <EnumValue Name="WinMDObj" />
```

Reply:
AppContainerExe is exe that lives in an "AppContainer", Windows Store app is a concept that existed in Windows 8 timeframe that doesn't exist. There is no better description other than "used by UWP projects"